### DISCUSSION TITLE
Are there any REST API's for Webauthn in Keycloak?

### DISCUSSION THREAD

**Discussion by Mohitkiran** [keycloak/keycloak#22893] [Q&A]
Hi folks,

I am trying to use keycloaks inbuilt webauth MFA for my native application and for that I need REST API's of webauth and I don't see anything on the documentation, can any one tell me if this is supported? If yes please point me in the right direction.

**Comment by justin-tay** [comment]
- https://github.com/keycloak/keycloak/discussions/22590
